lib/managedservicesplatform/clients/clients.go

```diff
@@ -0,0 +1,31 @@
+package clients
+
+import (
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/keepalive"
+)
+
+// NewRecommendedGRPCDialOptions provides additional networking options tailored
+// for connecting to typical MSP environments over gRPC (Cloud Run services
+// behind Cloudflare).
+func NewRecommendedGRPCDialOptions() []grpc.DialOption {
```

Comment:
```suggestion
func NewCloudflareGRPCDialOptions() []grpc.DialOption {
```

I am wondering if we should just be explicit in the naming. Would gRPC behave worse than it could if there is no Cloudflare in between using these settings?

Reply:
I don't think this is a Cloudflare problem, it's a Cloud Run problem

Reply:
But either way because I don't know exactly the source of the problems, just that we can't seem to maintain long-lived connections (Cloudflare is _supposed_ to allow it, but it's hard to tell), I think the more generic naming is probably safer